happy_birthder_script_tests.py

```diff
@@ -97,6 +466,9 @@ def main():
                       help='allows specifying admin username')
     parser.add_option('-p', '--password', dest='password',
                       help='allows specifying admin password')
+    parser.add_option('-w', '--wait', dest='wait',
+                      help='allows specifying time '
+                           'for waiting reminder\'s work(secs)')
```

Comment:
If you add the mandatory argument without default value you should notice this in README.